packages/ai-native/src/browser/mcp/tools/components/Terminal.tsx

```
@@ -63,19 +63,17 @@ export const TerminalToolComponent = memo((props: IMCPServerToolComponentProps)
                 <code>$ {args.command}</code>
               </p>
             </>
           )}
-          <div className={styles.command_title}>
-            <span>{localize('ai.native.mcp.terminal.output')}</span>
-          </div>
           {output ? (
             <div className={styles.command_content}>
-              <code>{stripAnsi(output.text)}</code>
+              <Icon icon='output' />
+              <code dangerouslySetInnerHTML={{ __html: computeAnsiLogString(output.text || '') }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)